Code/GraphMol/MolAlign/Wrap/rdMolAlign.cpp

```diff
@@ -75,10 +77,18 @@ PyObject *generateRmsdTransPyTuple(double rmsd, RDGeom::Transform3D &trans) {
       resData[itab + j] = tdata[itab + j];
     }
   }
-  PyObject *resTup = PyTuple_New(2);
+  PyObject *resTup = PyTuple_New(2 + (match ? 1 : 0));
   PyObject *rmsdItem = PyFloat_FromDouble(rmsd);
   PyTuple_SetItem(resTup, 0, rmsdItem);
   PyTuple_SetItem(resTup, 1, PyArray_Return(res));
+  if (match) {
```

Comment:
This code ends up returning the match as a tuple of 2-tuples. That's consistent with what the C++ code does, but the python versions of `GetSubstructMatch()` return a tuple of ints. I think that would be preferable here in order to maintain consistency within the Python code.
`Code/GraphMol/Wrap/substructmethods.h` has a `convertMatches()` function which you can use for this.

Reply:
That's not feasible. Since by design a `map` argument can be passed which may include `(probeAtomID, refAtomId)` partial matches between `prbMol` and `refMol`, in order to enable matching a larger molecule to a smaller molecule and vice versa (and subset thereof), the returned match can be itself a subset of both `prbMol` and `refMol` indices. Therefore, indices in both `prbMol` and `refMol` are required to enable a unequivocal mapping.